python/sdss_install/utils/module.py

```diff
@@ -79,8 +83,8 @@ def set_modules_lang(self):
 
     def set_tclsh(self):
         '''Set tclsh exec directory.'''
-        self.tclsh = str()
-        if self.ready:
+        self.tclsh = None
+        if self.ready and self.modules_lang['tcl']:
```

Comment:
In the method `set_modules_home()`, it seems that `self.modules_lang['tcl']' is always set to something. So does the `and self.modules_lang['tcl']` do anything in `if self.ready and self.modules_lang['tcl']:`?